qualabs/video-headline
# Setting variables on init db script

## Description:
Changed init.sh to use environment variables for the user and password. Updated docker-compose images and documentation alongside this change

## Related Issue (if appropriate):
https://app.asana.com/0/1205530699145161/1206048607296779/f

## Steps to test change:
- Tear down your video-headline environment
`docker-compose down -v   # This will stop and remove containers and volumes`
- Build it again
`docker-compose up --build  # This will rebuild and start the containers`
- VideoHeadline should be functional
